### ISSUE TITLE
quickstart unauth'd user returns 500 on creating services or credentials.

### ISSUE THREAD

**Issue by pnathan** [lyft/confidant#59]
Using the insecure quickstart configuration in the docs and version c7ca4e8206374c8c6e785b42eee732e23a8a15a5 on master:

New service, named "whiffleball", no creds...
```
confidant_1 |   File "/srv/confidant/confidant/authnz.py", line 156, in decorated
confidant_1 |     return f(*args, **kwargs)
confidant_1 |   File "/srv/confidant/confidant/authnz.py", line 106, in decorated
confidant_1 |     if check_csrf_token():
confidant_1 |   File "/srv/confidant/confidant/authnz.py", line 97, in check_csrf_token
confidant_1 |     if g.auth_type == 'kms':
confidant_1 |   File "/usr/local/lib/python2.7/dist-packages/werkzeug/local.py", line 343, in __getattr__
confidant_1 |     return getattr(self._get_current_object(), name)
confidant_1 | AttributeError: '_AppCtxGlobals' object has no attribute 'auth_type'
confidant_1 | 172.17.0.1 - - [18/Apr/2016:17:31:31 +0000] "PUT /v1/services/whiffleball HTTP/1.1" 500 - "-" "-"
```

New credentials, one row:

```
confidant_1 |   File "/srv/confidant/confidant/authnz.py", line 156, in decorated
confidant_1 |     return f(*args, **kwargs)
confidant_1 |   File "/srv/confidant/confidant/authnz.py", line 106, in decorated
confidant_1 |     if check_csrf_token():
confidant_1 |   File "/srv/confidant/confidant/authnz.py", line 97, in check_csrf_token
confidant_1 |     if g.auth_type == 'kms':
confidant_1 |   File "/usr/local/lib/python2.7/dist-packages/werkzeug/local.py", line 343, in __getattr__
confidant_1 |     return getattr(self._get_current_object(), name)
confidant_1 | AttributeError: '_AppCtxGlobals' object has no attribute 'auth_type'
confidant_1 | 172.17.0.1 - - [18/Apr/2016:18:02:40 +0000] "POST /v1/credentials HTTP/1.1" 500 - "-" "-"
```

This uses the `service.env` defined in the documentation.

**Comment by woodrow** [comment]
@pnathan are you sure this is master? I only see `auth_type` on 1.1, which is likely not ready for prime time.

```
$ git checkout master
Switched to branch 'master'
Your branch is up-to-date with 'origin/master'.
gstephenwoodrow@swoodrow-mbp121:~/lyft/confidant$ git pull
Already up-to-date.
$ git grep auth_type
$ git checkout 1.1
Switched to branch '1.1'
Your branch is up-to-date with 'origin/1.1'.
$ git pull
Already up-to-date.
$ git grep auth_type
confidant/authnz.py:97:    if g.auth_type == 'kms':
confidant/authnz.py:194:                    g.auth_type = 'kms'
confidant/authnz.py:221:                    g.auth_type = 'oauth'
confidant/authnz.py:251:                    g.auth_type = 'oauth'
```

**Comment by pnathan** [comment]
@woodrow  I'm afraid I stuffed my foot in my mouth - :embarassed: ; thought I was running on master. (although, you might  find this report useful for v1.1!)

**Comment by woodrow** [comment]
:) not a problem.

cc @ryan-lane for the 1.1 branch

**Comment by ryan-lane** [comment]
Yeah. I haven't tested the development/quickstart process in 1.1 yet. Thanks for the bug report! I'll make sure this is fixed before release.

**Comment by ryan-lane** [comment]
Fixed in 8b06fd6ecfe5a902897498aef84dadd44b1c9246

**Comment by ryan-lane** [comment]
This actually only exists in 1.1, so I'm going to close this as fixed :)
